task/01-strings-tasks.js

```diff
@@ -201,9 +196,34 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
-}
+  let rec = '';
+  for (let i = 1; i <= height; i++) {
+    for (let j = 1; j <= width; j++) {
+      if (j === 1 && i === 1) {
+        rec += '┌';
+      } else if (j === width && i === 1) {
+        rec += '┐';
+      } else if (j > 1 && j < width && i === 1) {
+        rec += '─';
+      } else if (j > 1 && j < width && i === height) {
+        rec += '─';
+      } else if (j === 1 && i === height) {
+        rec += '└';
+      } else if (j === width && i === height) {
+        rec += '┘';
+      } else if (j === 1 && i > 1 && i < height) {
+        rec += '│';
+      } else if (j === width && i > 1 && i < height) {
+        rec += '│';
+      } else {
+        rec += ' ';
+      }
+    }
 
+    rec += '\n';
+  }
```

Comment:
Acceptable, but can be done easier with string.[repeat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)